forc-plugins/forc-crypto/src/address.rs

```diff
@@ -4,11 +4,16 @@ use fuels_core::types::bech32::Bech32Address;
 use serde_json::json;
 use std::str::{from_utf8, FromStr};
 
+forc::cli_examples! {
+    [ Convert any valid address to all other supported formats => crypto "address fuel12e0xwx34nfp7jrzvn9mp5qkac3yvp7h8fx37ghl7klf82vv2wkys6wd523" ]
```

Comment:
```suggestion
    [ Convert an address to another format => crypto "address fuel12e0xwx34nfp7jrzvn9mp5qkac3yvp7h8fx37ghl7klf82vv2wkys6wd523" ]
```

I think it's implicit that we only convert a valid address and that it can be converted to any format that we support.

The API here is a bit unexpected imo because it seems like we use a keyword `address` and then any type of address. I would expect to pass a flag to tell it which format we want to convert it into but perhaps we supply any type of address and it prints all converted versions so that we don't have to supply any flags